registry.json

```diff
@@ -951,5 +951,20 @@
     "telegram": "edm0nd",
     "operatorName": "Edmond",
     "github": "spmedia"
+  },
+{
+    "ticker": "LOTUS",
+    "poolId": "0126c482283df9e693f28a25fb7e0ad890db5a269c695361069dfc81",
```

Comment:
I cannot find your pool.

Reply:
I fucked ip something 
Retired it will create again